From 9c444f5b9680ed5be75afcf8f728983f9c4be656 Mon Sep 17 00:00:00 2001
From: Adam Williamson <awilliam@redhat.com>
Date: Wed, 3 Jun 2020 15:05:54 -0700
Subject: [PATCH] HACK: nerf beginning_backslash test to pass with Python 3.9b4

This is a hack, not a real fix. We really want the empty lines
to be there, but the new parser in Python 3.9 treats an empty
line after a backslash like this as a syntax error:
https://bugs.python.org/issue40847
so the test cannot pass as intended on current Python 3.9. This
should be removed when Python is fixed.

Signed-off-by: Adam Williamson <awilliam@redhat.com>
---
 tests/data/beginning_backslash.py | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/tests/data/beginning_backslash.py b/tests/data/beginning_backslash.py
index 66c347d..73a1ba4 100644
--- a/tests/data/beginning_backslash.py
+++ b/tests/data/beginning_backslash.py
@@ -1,12 +1,5 @@
 \
-
-
-
-
-
 print("hello, world")
 
 # output
-
-
 print("hello, world")
-- 
2.26.2

